package.json

```diff
@@ -36,6 +36,12 @@
       "hash": "QmZvXudUXTc2G3o2SYPU7uQX1gzbeQkAZXJ6bnN8C4rcXE",
       "name": "go-libp2p-rpc",
       "version": "0.0.10"
+    },
+    {
+      "author": "hsanjuan",
+      "hash": "QmSNNKmo6LtoiSLjaVJxi76QFRfKJvJUME96EmtLEtFPdq",
+      "name": "cli",
+      "version": "2017.1.24"
```

Comment:
Yes, I can't find a better way to allow the package.json description of a package to take me back to the original time a package was gx'ed so it can be rebuilt again (shall ipfs ever lose the hash).

Actually I should set author to the original one. If I have an author, a project name and a date I can at least know when more or less this external dependency was packed in case I ever need to rebuild it or try to produce something similar.